atomix/cluster/src/main/java/io/atomix/raft/storage/system/Configuration.java

```diff
@@ -120,4 +121,8 @@ private static Collection<RaftMember> copyMembers(final Collection<RaftMember> m
     }
     return copied.build();
   }
+
+  public boolean hasMember(final MemberId member) {
+    return allMembers().stream().anyMatch(m -> m.memberId().equals(member));
```

Comment:
:thinking: I wonder if we really need to be creating a new set every time only to check for existence. I think it's fine here, refactoring this is out of scope, but this hints that perhaps we should always have the sets of members.

Reply:
I think it is fine because this is not invoked frequently. This is only during force configuration change, which doesn't happen frequently.

Reply:
Yeah, exactly. But it still hints that maybe we usually expect to have a set, and without knowing the implementation of `allMembers`, it could be we end up using it more often at some point. I just wanted to note in case we refactor around there again, we may want to clean up a bit our data model based on how we use it.